tmt/base.py

```diff
@@ -2072,6 +2072,17 @@ def __init__(self, data, name):
         except KeyError:
             self.log = []
 
+        # Handle alternative result interpretation
+        if interpret == "xfail":
+            self.result = {'fail': 'pass', 'pass': 'fail'}[self.result]
+        elif interpret in self._results:
+            self.result = interpret
+        elif interpret == "respect":
```

Comment:
Could we start with default value (a wee bit faster)?

Reply:
Yes, I'll rewrite it in the next commit.